source/interprocedural_analyses/taint/taintAnalysis.ml

```diff
@@ -66,32 +65,19 @@ let initialize_and_verify_configuration
           ~maximum_tito_depth
     |> TaintConfiguration.exception_on_error
   in
-  let () =
-    Statistics.performance
-      ~name:"Initialized and verified taint configuration"
-      ~phase_name:"Initializing and verifying taint configuration"
-      ~timer
-      ()
-  in
+  StepLogger.finish ~step_logger ~integers:[];
```

Comment:
This doesn't compile, it should be:
```suggestion
  StepLogger.finish ~integers:[] step_logger;
```
I will fix it myself so we don't take another week to merge this.
In the future, please run `make test` before submitting :)